conf/src/main/java/org/apache/nemo/conf/JobConf.java

```diff
@@ -156,6 +156,15 @@
   public final class JVMHeapSlack implements Name<Double> {
   }
 
+  /**
+   * Executor type:
+   * Choose lambda executor or default executor.
```

Comment:
@alapha23 Could you please list up possible types? (default , lambda , ...)

Reply:
Updated comments.